app/apollo/resolvers/cluster.js

```diff
@@ -444,6 +444,9 @@ const clusterResolvers = {
         const deletedResourceYamlHist = await models.ResourceYamlHist.updateMany({ org_id, cluster_id }, {$set: { deleted: true }}, { upsert: false });
         logger.info({req_id, user, org_id, cluster_id, deletedResourceYamlHist}, 'ResourceYamlHist soft-deletion complete');
 
+        // Allow graphQL plugins to retrieve more information
+        context.pluginContext = {registration: deletedCluster.registration, uuid: deletedCluster.cluster_id};
```

Comment:
Why the whole registration instead of just
```suggestion
        context.pluginContext = {name: deletedCluster.registration.name, uuid: deletedCluster.cluster_id};
```